### PR TITLE
Feature/refactor components motion keyframes

### PR DESCRIPTION
**Pull request by 3dyonic**
https://monday.monday.com/boards/2861766393/pulses/3260921059

#### Basic
- [ ] Used plop (`npm run plop`) to create a new component.
- [ ] PR has description.
- [ ] New component is functional and uses Hooks. 
- [ ] Component defines [`PropTypes`](https://reactjs.org/docs/typechecking-with-proptypes.html).
#### Style
- [ ] Styles are added to `NewComponent.modules.scss` file inside of the `NewComponent` folder.
- [ ] Component uses CSS Modules.
- [ ] Design is compatible with [Monday Design System](https://design.monday.com/).
#### Storybook
- [ ] Stories were added to `/src/NewComponent/__stories__/NewComponent.stories.js` file.
- [ ] Stories include all flows of using the component.
#### Tests
- [ ] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

### CODE REVIEW

File: src/components/Counter/Counter.scss [hunk 1]
```diff
@@ -62,7 +63,7 @@
       position: relative;
       opacity: 0;
       transform: translateY(-15px);
-      transition: transform var(--motion-productive-long) var(--motion-timing-enter), opacity var(--motion-productive-short);
+      transition: transform var(--motion-productive-long) var(--motion-timing-exit), opacity var(--motion-productive-short);
```
Comment (SergeyRoyt) [inline src/components/Counter/Counter.scss:66]: Why this was changed?

File: src/components/Toast/__stories__/toast.stories.mdx [hunk 2]
```diff
@@ -248,7 +248,7 @@ After a user has done an action, provide feedback to close the loop. For example
             type={Toast.types.POSITIVE}
             actions={actions}
             onClose={onCloseCallback}
-            autoHideDuration={5000}
+            autoHideDuration={50000000}
```
Comment (SergeyRoyt) [inline src/components/Toast/__stories__/toast.stories.mdx:251]: I guess this has been changed for test purposes - fix needed

File: src/components/Toast/__stories__/toast.stories.scss [hunk 3]
```diff
@@ -1,6 +1,7 @@
 @import "../../../styles/themes";
 
 .monday-storybook-toast {
+  border: 1px solid red;
```
Comment (SergeyRoyt) [inline src/components/Toast/__stories__/toast.stories.scss:4]: Fix needed

File: src/components/Toast/__stories__/toast.stories.scss [hunk 4]
```diff
@@ -35,3 +36,9 @@
     width: 390px;
   }
 }
+
+// overriding margin for storybook wrapper
+.monday-storybook_multiple-story-elements-wrapper .monday-style-toast{
+  margin-right:auto;
+  margin-left:auto;
```
Comment (SergeyRoyt) [inline src/components/Toast/__stories__/toast.stories.scss:43]: What is this about? What have u fixed?

File: src/styles/_keyframes.scss [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+// MIXIN: Keyframe
+// This mixin defines a keyframe definition using @content.
+// @argument: $animation-name
+// This mixin is used to enable reuse for local keyframes content
+
+@mixin keyframe($animation_name) {
+  @keyframes #{$animation_name} {
+    @content;
+  }
+}
```
Comment (SergeyRoyt) [inline src/styles/_keyframes.scss:25]: Love it 👍

File: src/components/Counter/Counter.scss (hunk 1 above)
Comment (3dyonic) [reply src/components/Counter/Counter.scss:66]: its exit timing... and exit transition.

File: src/components/Toast/__stories__/toast.stories.mdx (hunk 2 above)
Comment (3dyonic) [reply src/components/Toast/__stories__/toast.stories.mdx:251]: sure thanks.

File: src/components/Toast/__stories__/toast.stories.scss (hunk 4 above)
Comment (3dyonic) [reply src/components/Toast/__stories__/toast.stories.scss:43]: the initial toast had transform: transitionX(-50%) , i remove it and used margin auto,but there is a legacy selector that Hadas wrote only in story book that had Heighr specificity. 
This solves this.

**Review by SergeyRoyt — APPROVED** [review]

**Pull request merged**